ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -89,6 +89,18 @@ Item {
             chatSectionModule: root.rootStore.chatCommunitySectionModule
         }
 
+        readonly property string linkPreviewNotificationTemplate: `%1<a style="text-decoration:none" href="#${Constants.appSection.profile}/${Constants.settingsSubsection.messaging}">%2</a>.`
+
+        readonly property string linkPreviewEnabledNotification: linkPreviewNotificationTemplate
+                                                                    .arg(qsTr("Link previews will be shown for all sites. You can manage link previews in "))
```

Comment:
WDYT about doing the opposite? Something like qsTr("Link previews will be shown for all sites. You can manage link previews in %1Settings%2.").arg(startAnchor).arg(endAnchor)

Reply:
I was thinking more about:
`qsTr("You can manage link previews in %1").arg("<a>" + qsTr("Settings") + "</a>")`

Reply:
> It's not nice to cut the sentence like that :/

Then I misunderstood this comment. 😄 

Why I think it's not nice to cut it like that is because it makes it hard to do an appropriate translation to languages where the semantics is important. This version has the entire context for an appropriate translation `qsTr("Link previews will be shown for all sites. You can manage link previews in %1Settings%2.")`

Reply:
But then the translator is free to exchange the order of the placeholders which would break the text (eg `%2Settings%1`); not a great solution either imo

Reply:
Ohh, right! Let's go for your version then and add context to the translation:

`qsTr("You can manage link previews in %1", "go to settings page").arg("<a>" + qsTr("Settings", "settings page") + "</a>")`

Reply:
Fixed